SWIFT_PACKAGE_MANAGER.md

```diff
@@ -0,0 +1,92 @@
+# Swift Package Manager Instructions
+
+Support for Swift Package Manager was introduced in Braintree iOS v5.
+
+* [General Instructions](#general-instructions)
+* [Binary Dependencies](#binary-dependencies)
+* [BraintreeDataCollector](#braintreedatacollector)
+* [BraintreeThreeDSecure](#braintreethreedsecure)
+
+### General Instructions
+
+To add the `Braintree` package to your Xcode project, select File > Swift Packages > Add Package Dependency and enter `https://github.com/braintree/braintree_ios` as the repository URL. Tick the checkboxes for the specific Braintree libraries you wish to include.
```

Comment:
Nitpick, but it looks like there is some inconsistency between italics, quotes, and plain text to note selections or buttons to click (for example this line vs line 31 vs line 57)

Reply:
Got it - will clean this up now